packages/aragon-wrapper/src/cache/index.js

```diff
@@ -17,6 +17,52 @@ import * as configurationKeys from '../configuration/keys'
  *        Require the cache to downgrade to localstorage even if IndexedDB is available
  */
 export default class Cache {
+  static cacheTracker
+
+  static initCacheTracker = async () => {
+    if(!Cache.cacheTracker) {
+      Cache.cacheTracker = new Cache(configurationKeys.CACHE_TRACKER_PREFIX)
+      await Cache.cacheTracker.init()
+    }
+  }
+
+  static getSavedCaches = async () => {
+    const savedCaches = await Cache.cacheTracker.get(configurationKeys.CACHE_TRACKER_CACHES_KEY)
+    return Array.from(new Set(savedCaches)).filter(c => !!c)
+  }
+
+  static trackNewCache = async (prefix) => {
+    if(!Cache.cacheTracker) {
+      await Cache.initCacheTracker()
+    }
+
+    const savedCaches = await this.getSavedCaches()
+    if(savedCaches[0]) {
+      await Cache.cacheTracker.set(configurationKeys.CACHE_TRACKER_CACHES_KEY, [...savedCaches, prefix])
+    } else {
+      await Cache.cacheTracker.set(configurationKeys.CACHE_TRACKER_CACHES_KEY, [prefix])
+    }
+  }
+
+  static clearAllCaches = async () => {
```

Comment:
I would use for all those methods the "normal" syntax e.g.:
``static async clearAllCaches() {...}